src/Components/Steps/Expenses/Expenses.tsx

```diff
@@ -0,0 +1,323 @@
+                  <CloseButton handleClose={() => remove(index)} />
+                </div>
+              )}
+              <Stack className={index % 2 === 0 ? 'expense-section' : ''}>
+                <QuestionQuestion>
+                  {index === 0 ? (
+                    <div className="first-expense-q-padding">
+                      <FormattedMessage
+                        id="questions.hasExpenses-a"
+                        defaultMessage="What type of expense has your household had most recently?"
+                      />
+                    </div>
+                  ) : (
+                    <FormattedMessage
+                      id="expenseBlock.createExpenseBlockQuestions-questionLabel"
+                      defaultMessage="If you have another expense, select it below."
+                    />
+                  )}
+                </QuestionQuestion>
+              </Stack>
+              <FormControl
+                sx={{ m: 1, minWidth: '13.125rem', maxWidth: '100%' }}
+                error={!!errors.expenses?.[index]?.expenseSourceName}
+              >
+                <InputLabel id={`expense-type-label-${index}`}>
+                  <FormattedMessage
+                    id="expenseBlock.createExpenseDropdownMenu-expenseTypeInputLabel"
+                    defaultMessage="Expense Type"
+                  />
+                </InputLabel>
+                <Controller
+                  name={`expenses.${index}.expenseSourceName`}
+                  control={control}
+                  render={({ field }) => (
+                    <>
+                      <Select
+                        {...field}
+                        labelId={`expense-type-label-${index}`}
+                        id={`expenses.${index}.expenseSourceName`}
+                        label={
+                          <FormattedMessage
+                            id="expenseBlock.createExpenseDropdownMenu-expenseTypeSelectLabel"
+                            defaultMessage="Expense Type"
+                          />
+                        }
+                        sx={{ backgroundColor: '#fff' }}
+                      >
+                        {createExpenseMenuItems(expenseOptions)}
+                      </Select>
+                      {!!errors.expenses?.[index]?.expenseSourceName && (
+                        <FormHelperText sx={{ marginLeft: 0 }}>{renderExpenseSourceHelperText()}</FormHelperText>
+                      )}
+                    </>
+                  )}
+                />
+              </FormControl>
+              <div className="expense-margin-bottom">
+                <QuestionQuestion>
+                  <FormattedMessage
+                    id="expenseBlock.createExpenseAmountTextfield-questionLabel"
+                    defaultMessage="How much is this expense every month "
+                  />
+                  {getExpenseSourceLabel(expenseOptions, selectedExpenseSource)}
```

Comment:
If you make the change to `getExpenseSourceLabel` above.
```suggestion
                  {getExpenseSourceLabel(selectedExpenseSource)}
```

Reply:
The `expenseOptions` dictionary is needed to render the correct label.